packages/api-svc/src/service.ts

```diff
@@ -171,7 +212,9 @@ export class Service {
 		}
 		globalLogger = this.logger = logger;
 
-		if(!tokenHelper){
+		if (!tokenHelper && bareboneStartup) {
```

Comment:
This code should only check if an implementation was provided, if not use the default. 
For production, the Service.Start() should be called with empty params.
For tests, the Service.Start() should inject the mocks via its params.
(same for others below)